runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -519,20 +538,21 @@ MM_CopyForwardScheme::postProcessRegions(MM_EnvironmentVLHGC *env)
 			Assert_MM_false(region->_reclaimData._shouldReclaim);
 
 			/* we do not count tails, only regions that we acquired as free */
-			if (region->getLowAddress() == region->_copyForwardData._survivorBase) {
+			if (region->isFreshSurvivorRegion()) {
```

Comment:
comment above should be updated. 'tail' should probably be used anywhere, anymore, right?